style.css

```diff
@@ -74,3 +73,165 @@ main {
   border-radius: 50%;
   color: blue;
 }
+
+ul {
+  list-style-type: none;
+}
+
+/* Recent Work Session */
+
+.container-recent {
+  display: grid;
+  margin-left: 10px;
+  margin-right: 10px;
+  margin-bottom: 52px;
+}
+
+.grid-project {
+  flex-direction: column;
+  text-align: center;
+  padding-left: 1.5px;
+  padding-right: 1.5px;
+  padding-bottom: 1.5px;
+  padding-top: 252px;
+}
+
+.container-recent > .grid-project {
+  background-color: #d0d9d4;
+  border-radius: 8px;
+  margin-top: 10px;
+}
+
+.multi-post {
+  background-color: white;
+  border-radius: 8px;
+  line-height: 8px;
+  padding-top: 12px;
+}
+
+.multi-text-color {
+  font-weight: 600px;
+  font-size: 24px;
+  color: #3a4a42;
+}
+
+.see-project {
+  background-color: #36b37f;
+  padding: 8px;
+  border-color: #36b37f;
+  border-radius: 4px;
+  color: white;
+  cursor: pointer;
+  margin-bottom: 16px;
+}
+
+.see-project:hover {
+  background-color: #36b37f;
+  cursor: pointer;
+  transform: scale(0.9);
+  border-radius: 16px;
+  box-shadow: 0 8px rgba(54, 179, 127, 0.24);
+}
+
+.see-project:active {
+  background-color: #008552;
+  transform: scale(0.9);
+}
+
+.language {
+  flex-wrap: wrap;
+  column-gap: 10px;
+  margin-top: 8px;
+  justify-content: center;
+}
```

Comment:
- [ ] in order to comply with the requirement the previous CR mentioned regarding the alignment of the tech-stack inside the projects section, you should add `padding: 0;` to the `language` class

Reply:
thanks, done!